wire/cli/src/cli.rs

```diff
@@ -78,7 +78,7 @@ pub enum Commands {
 
         /// List of literal node names or `@` prefixed tags.
         #[arg(short, long, value_name = "NODE | @TAG", num_args = 1..)]
-        on: Vec<ApplyTarget>,
+        on: Option<Vec<ApplyTarget>>,
```

Comment:
I think it would be cool to use `--on -` instead of omitting the option

Reply:
iirc that's not possible

Reply:
> iirc that's not possible

I mean it is of course but we would have to implement custom logic for that.